SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -107,16 +105,26 @@ void FSpatialGDKEditorToolbarModule::PreUnloadCallback()
 {
 	if (bStopSpatialOnExit)
 	{
-		StopRunningStack();
+		LocalDeploymentManager->TryStopLocalDeployment();
 	}
 }
 
 void FSpatialGDKEditorToolbarModule::Tick(float DeltaTime)
 {
-	if (SpatialOSStackProcessID != 0 && !FPlatformProcess::IsApplicationRunning(SpatialOSStackProcessID))
+	// Bind the play button delegate to starting a local spatial deployment.
+	if (!GEditor->TryStartSpatialDeployment.IsBound())
```

Comment:
Doing this ever tick feels bad. Can't we do it on init?

Reply:
So I've had trouble finding a good place for this. It's issues with module loading orders. Init does not work as accessing `GEditor->TryStartSpatialDeployment` at this time causes a crash. Any other ideas?